src/OmnichannelChatSDK.ts

```diff
@@ -1718,6 +1767,16 @@ class OmnichannelChatSDK {
 
         if (this.liveChatVersion === LiveChatVersion.V2) {
             try {
+                if (this.isAMSClientAllowed === false) {
+                    this.scenarioMarker.failScenario(TelemetryEvent.DownloadFileAttachment, {
```

Comment:
`failScenario()` is not needed anymore

Reply:
I rather have a pre check instead of continue and wait for the fail